src/utilities/queryStringToSearchCriteria.js

```diff
@@ -236,8 +236,8 @@ export const queryStringToSearchCriteria = async (
 
 	// TODO: This should only be allowed on the results page
 	if (query['pn']) {
-		const pageNum = parseInt(query['pn']);
-		if (!Number.isNaN(pageNum) && pageNum > 0) {
+		let pageNum = parseInt(query['pn']);
```

Comment:
Is there a particular reason this is a `let` rather than a `const`?

Reply:
Updated.  Changed originally for logic since removed